### DISCUSSION TITLE
Spacing between the columns

### DISCUSSION THREAD

**Discussion by deltapapa01** [templaza/astroid-framework#386] [General]
Hello,
I have a problem that I can not get solved.
I want to have a white space between each colum. My layout:

![image](https://github.com/templaza/astroid-framework/assets/19833064/bf5de9c0-3ed9-4f5c-a34e-4d0bbae2df97)

My frontend looks like this at the moment:

![image](https://github.com/templaza/astroid-framework/assets/19833064/e877844c-4199-4a31-b7a0-a68030ee99c2)

But no matter what I set in margin and padding, I can't get a white space between the columns.

This is how it looked on my old site with JSN template:

![image](https://github.com/templaza/astroid-framework/assets/19833064/9e2d0946-74ef-49e2-9261-bf466cf83ea4)

Does anyone have an idea how da can set a distance?

WBR,
deltapapa

**Comment by sonvnn** [comment]
Hi,

Default gutter width is `1.5rem` https://getbootstrap.com/docs/5.3/layout/gutters/#change-the-gutters

You can use sass overrides it in Theming options
![Screen Shot 2023-09-06 at 10 08 05](https://github.com/templaza/astroid-framework/assets/3204103/e3bba063-c115-4d3c-af39-0cd56712fede)

Best Regards,
Sonny
